addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -372,7 +372,8 @@ class ApplyConfiguratorScreen extends Component {
             // Here the website service goToWebsite method is not used because
             // the web client needs to be reloaded after the new modules have
             // been installed.
-            window.location.replace(`/web#action=website.website_preview&website_id=${encodeURIComponent(resp.website_id)}&enable_editor=1&with_loader=1`);
+            // Todo in master: remove the no_edit_on_mobile parameter.
+            window.location.replace(`/web#action=website.website_preview&website_id=${encodeURIComponent(resp.website_id)}&enable_editor=1&with_loader=1&no_edit_on_mobile=1`);
```

Comment:
@mart-e The reason I think we can override ci security here is that the `window.location.replace` already existed before (it has been added by [this commit](https://github.com/odoo/odoo/commit/caefcb8590301a0ebbe947a875003b33ce7d537a#diff-d333448efc33dffb2482b17a4facd3d8a8d8e678430513aa00e9f4eb33d165e8R387) from [this PR that had the override ci security](https://github.com/odoo/odoo/pull/89223#issuecomment-1163762100)) and I'm just adding a parameter that doesn't even come from the user. Let me know if I can do anything :+1:

Reply:
> already existed before

That's not a valid justification, always a good idea to recheck.

>  I'm just adding a parameter that doesn't even come from the user

That is the kind of justification we want :)

We catch on `window.location` because we don't want to allow random users to redirect users to random pages or to inject javascript in the URL. Here it is not possible becaus only `resp.website_id` is variable and it is wrapped with `encodeURIComponent`.

@robodoo override=ci/security

Reply:
Thanks for the explanation :+1: